src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -22,28 +21,38 @@ class EnrollmentHistory extends React.Component {
     */
 
     // History explanation box
+    let historyExplanationBox;
     const benefitMaybePending =
-      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement > 0) &&
-      enrollmentData.usedEntitlement === 0;
+      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement.months > 0) &&
```

Comment:
Yes, although the only thing I think we need to change is the `originalEntitlement` check:
```
enrollmentData.originalEntitlement.months > 0 || enrollmentData.originalEntitlement.days > 0
```
I think this can be an OR instead of an AND because all that matters is that some amount of time is in `originalEntitlement`. I could imagine a scenario where they have some number of months, but 0 days, and this would still could as having an original entitlement.